src/main/webapp/go.js

```diff
@@ -48,10 +66,87 @@ function getHunt() {
       puzzleArr.push(mssg.items[i].riddle.puzzle);
       destArr.push(mssg.items[i].name);
     }
+    createMap();
     updateToCurrentState(destIndex);
   });
 }
 
+/**
+ * Add a marker to the map at the specified location.
+ * @param {Double} destLat: latitude of location.
+ * @param {Double} destLng: longitude of location.
+ * @param {String} destName: Name of location.
+ */
+function addMarkerToMap(destLat, destLng, destName) {
+  const coord = new google.maps.LatLng(destLat, destLng);
+  const marker = new google.maps.Marker({
+    position: coord,
+    title: destName,
+  });
+  marker.setMap(map);
+}
+
+/**
+ * Creates a map and adds it to the page.
+ */
+function createMap() {
+  map = new google.maps.Map(
+      document.getElementById(MAP_DISPLAY),
+      // Centered at GooglePlex (updated below).
+      {center: {lat: 37.422, lng: -122.084}, zoom: 7},
+  );
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(function(position) {
+      const pos = {lat: position.coords.latitude,
+        lng: position.coords.longitude};
+      addMarkerToMap(position.coords.latitude, position.coords.longitude,
+          'Your current location');
+      map.setCenter(pos);
+    }, function() {
+      updateMessage(MAP_MSSG_DISPLAY, 'Error: I can\'t find your location.');
+    });
+  } else {
+    updateMessage(MAP_MSSG_DISPLAY, 'Error: Your browser doesn\'t' +
+        'support geolocation.');
+  }
+}
+
+/** PR #2
```

Comment:
What is this comment for?

Reply:
oops -- I initially added comments to decide how to break up the pull requests, and forgot to take them out.